frame/vesting/src/lib.rs

```diff
@@ -76,36 +82,39 @@ type MaxLocksOf<T> =
 	<<T as Config>::Currency as LockableCurrency<<T as frame_system::Config>::AccountId>>::MaxLocks;
 
 const VESTING_ID: LockIdentifier = *b"vesting ";
+const LOG_TARGET: &'static str = "runtime::vesting";
+
+// A value placed in storage that represents the current version of the Vesting storage.
+// This value is used by `on_runtime_upgrade` to determine whether we run storage migration logic.
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+enum Releases {
+	V0,
+	V1,
+}
 
-/// Struct to encode the vesting schedule of an individual account.
-#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug)]
-pub struct VestingInfo<Balance, BlockNumber> {
-	/// Locked amount at genesis.
-	pub locked: Balance,
-	/// Amount that gets unlocked every block after `starting_block`.
-	pub per_block: Balance,
-	/// Starting block for unlocking(vesting).
-	pub starting_block: BlockNumber,
+impl Default for Releases {
+	fn default() -> Self {
+		Releases::V0
```

Comment:
The default should be v1 now right?

Anyone who uses this pallet to start with will be on V1

Reply:
Actually dont implement default here, instead you need to write custom logic for default in the storage below

Reply:
Follow the template in staking:

https://github.com/paritytech/substrate/blob/master/frame/staking/src/lib.rs#L748